pkgs/development/python-modules/pythonocc-core/default.nix

```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation rec {
     owner = "tpaviot";
     repo = "pythonocc-core";
     tag = version;
-    hash = "sha256-45pqPQ07KYlpFwJSAYVHbzuqDQTbAvPpxReal52DCzU=";
+    hash = "sha256-0o2PQEN0/Z7FUPZEo2HxFFa+mN2bZnYI++HVu4ONpNA=";
```

Comment:
Any reason this was changed in a commit removing a maintainer

Reply:
oops, good eyes - that was accidentally committed from #388956 . now fixed.